### ISSUE TITLE
TDSM Core Module Plugin Crash on /spawnboss <BOSS>

### ISSUE THREAD

**Issue by D1G1T4L3CH0** [DeathCradle/Terraria-s-Dedicated-Server-Mod#98]
Using command `/spawnboss <BOSS>` results in a core plugin crash. Happens with multiple bosses. Some that I have tested so far are: wof, plantera
```
3/11/2015 1:52:40 AM Pool> 127.0.0.1:53147 @ 254: ENTER D1G1T4L3CH0
3/11/2015 1:52:51 AM Pool> D1G1T4L3CH0 sent command: /spawnboss wof
3/11/2015 1:52:51 AM Pool> Wall of Flesh was summoned by D1G1T4L3CH0
3/11/2015 1:52:51 AM Pool> Plugin TDSM Core Module crashed in hook player-chat
3/11/2015 1:52:51 AM Pool> System.NullReferenceException: Object reference not set to an instance of an object.
   at tdsm.core.Entry.SummonBoss(ISender sender, ArgumentList args)
   at tdsm.api.Command.CommandInfo.Run(ISender sender, ArgumentList args)
   at tdsm.api.Command.CommandParser.ParseAndProcess(ISender sender, String line)
   at tdsm.api.Command.CommandParser.ParsePlayerCommand(Player player, String line)
   at tdsm.core.Entry.OnChat(HookContext& ctx, PlayerChat& args)
   at tdsm.api.Plugin.HookPoint`1.Invoke(HookContext& context, T& arg)
```

**Comment by D1G1T4L3CH0** [comment]
This still happens.

**Comment by DeathCradle** [comment]
Should be fixed now in 486f201a3235a2145d38a9645e5dbbd117130ae5
